spec/lib/miq_automation_engine/service_methods/miq_ae_service_manageiq-providers-ansible_tower-configuration_manager-playbook_spec.rb

```diff
@@ -0,0 +1,12 @@
+module MiqAeServiceManageIQProvidersAnsibleTowerConfigurationManagerPlaybookSpec
```

Comment:
I see. Removed them now. (I basically copy this from other similar tests, so the others would need some cleanup)